homeassistant/components/shelly/utils.py

```diff
@@ -72,12 +73,13 @@ def get_number_of_channels(device: BlockDevice, block: Block) -> int:
 def get_block_entity_name(
     device: BlockDevice,
     block: Block | None,
-    description: str | None = None,
+    description: str | DeviceClassName | None = None,
```

Comment:
I don't understand the purpose of this change

Reply:
`get_block_entity_name` and `get_rpc_entity_name` are called with `EntityDescription.name`